### PR TITLE
Fix issue with DSCP setting. Add additional validation for everflow input parameters.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

**Comment by stcheng** [comment]
checking

File: orchagent/mirrororch.cpp [hunk 1]
```diff
@@ -327,8 +369,10 @@ bool MirrorOrch::activateSession(const string& name, MirrorEntry& session)
     attr.value.u8 = MIRROR_SESSION_DEFAULT_IP_HDR_VER;
     attrs.push_back(attr);
 
+    // TOS value format is the following:
+    // DSCP 6 bits | ECN 2 bits
     attr.id =SAI_MIRROR_SESSION_ATTR_TOS;
```
Comment (stcheng) [inline orchagent/mirrororch.cpp]: have one space between the '=' and the attribute name.
also fix the other places.

Comment (stcheng) [inline orchagent/mirrororch.cpp]: question: currently we don't have ecn settings?

File: orchagent/mirrororch.cpp [hunk 2]
```diff
@@ -91,49 +121,61 @@ void MirrorOrch::createEntry(const string& key, const vector<FieldValueTuple>& d
...
-        {
-            entry.srcIp = fvValue(i);
-            if (!entry.srcIp.isV4())
+        try {
+            if (fvField(i) == MIRROR_SESSION_SRC_IP)
+            {
+                entry.srcIp = fvValue(i);
+                if (!entry.srcIp.isV4())
+                {
+                    SWSS_LOG_ERROR("Unsupported version of sessions %s source IP address\n", key.c_str());
```
Comment (stcheng) [inline orchagent/mirrororch.cpp]: just remove all the '\n'

File: orchagent/mirrororch.cpp (hunk 1 above)
Comment (oleksandrivantsiv) [inline orchagent/mirrororch.cpp]: Yes. ECN was requested in arc spec only DSCP

**Pull request merged**